File: core/Dtos/CandidateAssessedDto.cs
namespace core.Dtos
{
     public class CandidateAssessedDto
    {
        public CandidateAssessedDto()
        {
        }

        public int Id {get; set;}
        public int OrderItemId { get; set; }
        public string CategoryRef {get; set;}
        public string CategoryName {get; set;}
        public bool requireInternalReview {get; set;}
        public string CustomerName {get; set;}
        public int CandidateId {get; set;}
        public string CandidateName {get; set;}
        public int ApplicationNo {get; set;}
        public string AgentName {get; set;}
        public string CityName { get; set; }
        public int AssessedById { get; set; }
        public string AssessedByName {get; set;}
        public string AssessedResult {get; set;}
        public string checklistedByName {get; set;}
        public DateTime ChecklistedOn {get; set;}
        public string Charges {get; set;}
        public int CvRefId {get; set;}
        public DateTime AssessedOn { get; set; }
        public string Remarks { get; set; }

    }
}

File: client2/src/app/shared/dtos/hr/candidateAssessedDto.ts
export interface ICandidateAssessedDto
{
     id: number;
     checked: boolean;
     orderItemId: number;
     orderItemRef: string;
     categoryName: string;
     applicationNo: number;
     customerName: string;
     requireInternalReview: boolean;
     candidateId: number;
     candidateName: string;
     assessedById: number;
     checklistedByName: string;
     checklistedOn: Date;
     charges: string;
     assessedByName: string;
     assessedResult: string;
     assessedOn: Date;
     cvRefId: number;
     remarks: string;
}

export class CandidateAssessedDto implements ICandidateAssessedDto
{
     id= 0;
     checked= false;
     orderItemId= 0;
     orderItemRef= '';
     categoryName= '';
     applicationNo= 0;
     customerName= '';
     requireInternalReview= false;
     candidateId= 0;
     candidateName= '';
     assessedById= 0;
     checklistedByName= '';
     checklistedOn: Date = new Date('1900-01-01');
     charges= '';
     assessedByName= '';
     assessedResult= '';
     assessedOn: Date = new Date('1900-01-01');
     cvRefId= 0;
     remarks= '';
}